CMakeLists.txt

```diff
@@ -39,6 +39,12 @@ else()
   FetchContent_MakeAvailable(mlx)
 endif()
 
+if(WIN32)
+   find_package(dlfcn-win32 REQUIRED)
+   message(STATUS "dlfcn-win32 lib " ${dlfcn-win32_LIBRARIES})
+   message(STATUS "dlfcn-win32 include " ${dlfcn-win32_INCLUDE_DIRS})
+endif()
```

Comment:
Shouldn't building MLX already handle this?

Reply:
For some reason it failed without it: https://buildkite.com/julialang/yggdrasil/builds/15057#01936cb8-239f-419a-868b-3d225dc261cb/6-20056 (wait for the giant log to load when clicking the link)

Fixed by including it: https://buildkite.com/julialang/yggdrasil/builds/15058#01936cc7-0e98-4f21-b318-64255454c6f0/6-19941 (wait for the giant log to load when clicking the link)

Reply:
But the static linking with dlfcn-win32 introduced with ml-explore/mlx#1690 will likely make this superfluous.